### DISCUSSION TITLE
Starter project in Factor?

### DISCUSSION THREAD

**Discussion by mrjbq7** [BattlesnakeOfficial/feedback#259] [General]
I noticed a [series of videos on building Battlesnake](https://re.factorcode.org/2024/09/battlesnake.html) in the [Factor programming language](https://factorcode.org).

But I also noticed that it is not listed on the [Starter Projects](https://docs.battlesnake.com/starter-projects) documentation.

Does anyone know where the code for it is?  It would be fun to link to for the Factor programming community.

**Comment by mrjbq7** [comment]
Nevemrind, found the original one:

https://github.com/joenash/factor-snake

And also

https://github.com/gifti258/factor-battlesnake

**Comment by xtagon** [comment]
> Does anyone know where the code for it is? 

@mrjbq7 The code for the starter projects is in https://github.com/BattlesnakeOfficial/docs -- Feel free to add the ones you found to the list, under "Community Projects" if they are not official or up to date. Make sure to read [the contributing guide](https://github.com/BattlesnakeOfficial/docs/blob/main/CONTRIBUTING.md) in the docs repo.
